### PR TITLE
Feature/login style

### PR DESCRIPTION
**Pull request by Eddie98Lopez**
## Description

The previous login page was not responsive to viewport width changes. This change works to fix that with the use of media queries for specific elements for tablet and mobile views. These changes were also made to match the approved wireframes discussed in the stakeholder meetings. These changes are strictly stylistic and should not impact functionality. 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have removed unnecessary comments/console logs from my code
- [ ] I have made corresponding changes to the documentation if necessary (optional)
- [ ] My changes generate no new warnings
- [ ] I have checked my code and corrected any misspellings
- [ ] No duplicate code left within changed files
- [ ] Size of pull request kept to a minimum
- [ ] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by salvare736 — APPROVED** [review]
These changes look great to me. Good work!

**Review by kmcervan — APPROVED** [review]
Agreed, it looks good.

File: src/components/pages/Login/_LoginContainerStyled.less [hunk 1]
```diff
@@ -55,10 +155,13 @@
 
     #okta-sign-in.auth-container.main-container {
       font-family: 'Lato' sans-serif;
+      width:100%;
       margin: 0;
       border: none;
       box-shadow: none;
       color: @font-color;
+      background:none;
+      //transform: translatex(7%);
```
Comment (elCarlosSantiago) [inline src/components/pages/Login/_LoginContainerStyled.less:164]: Lets get rid of this comment

File: src/reset.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
```
Comment (elCarlosSantiago) [inline src/reset.css:44]: missing a new line character here, a nitpick though. Tooling like prettier will take care of that.

File: src/components/pages/Login/_LoginContainerStyled.less [hunk 3]
```diff
@@ -1,32 +1,131 @@
...
+
+.bgImgContain{
+  width:100%;
+  height:100%;
+  position:fixed;
+  z-index: -3;
+
+}
+
+
```
Comment (elCarlosSantiago) [inline src/components/pages/Login/_LoginContainerStyled.less:70]: lets try to get consistent spacing between classes!

**Review by elCarlosSantiago — COMMENTED** [review]
couple of comments here and there but overall looks great!

**Pull request merged**